src/Structs/Script.php

```diff
@@ -11,4 +11,23 @@ protected function tag(): string
     {
         return 'script';
     }
+
+    public static function defaults(Struct $struct): void
+    {
+        $struct->addAttribute('type', 'application/javascript');
+    }
```

Comment:
I would not make this change since the [HTML5 spec recommends omitting the `type` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attr-type)

Reply:
Ah okay, I can take it out. Should I remove it and we keep only the `src`? I've got a few more things I wonder if they might be a good additional. I'll open an issue to discuss.

Reply:
You can keep the setter methods

Reply:
Done @romanzipp